src/classic/components/InfoCols/InfoCols.stories.tsx

```diff
@@ -0,0 +1,195 @@
+import { Meta, StoryFn } from "@storybook/react"
+import InfoCols from "./InfoCols"
+import { InfoColsProps } from "~/common"
+
+export default {
+  title: "Classic/Components/InfoCols",
+  component: InfoCols,
+  argTypes: {},
+} as Meta
+
+// Template for stories
+const Template: StoryFn<InfoColsProps> = (args) => <InfoCols {...args} />
+
+// Default in CMS is 3 infoboxes
+export const Default = Template.bind({})
+Default.args = {
+  sectionIdx: 0,
+  title: "Infobar title",
+  subtitle: "subtitle",
+  buttonLabel: "Button text",
+  buttonUrl: "https://google.com",
+  infoBoxes: [
+    {
+      title: "Infobox title",
+      description: "Infobox description",
+    },
+    {
+      title: "Infobox title",
+      description: "Infobox description",
+    },
+    {
+      title: "Infobox title",
+      description: "Infobox description",
+    },
+  ],
+}
+
+export const GrayBackground = Template.bind({})
+GrayBackground.args = {
+  sectionIdx: 1,
+  title: "Infobar title",
+  subtitle: "subtitle",
+  buttonLabel: "Button text",
+  buttonUrl: "https://google.com",
+  infoBoxes: [
+    {
+      title: "Infobox title",
+      description: "Infobox description",
+    },
+    {
+      title: "Infobox title",
+      description: "Infobox description",
+    },
+    {
+      title: "Infobox title",
+      description: "Infobox description",
+    },
+  ],
+}
+
+export const NoButton = Template.bind({})
+NoButton.args = {
+  sectionIdx: 0,
+  title: "Infobar title",
+  subtitle: "subtitle",
+  infoBoxes: [
+    {
+      title: "Infobox title",
+      description: "Infobox description",
+    },
+    {
+      title: "Infobox title",
+      description: "Infobox description",
+    },
+    {
+      title: "Infobox title",
+      description: "Infobox description",
+    },
+  ],
+}
+
+export const OneInfoBox = Template.bind({})
+OneInfoBox.args = {
+  sectionIdx: 0,
+  title: "Infobar title",
+  subtitle: "subtitle",
+  buttonLabel: "Button text",
+  buttonUrl: "https://google.com",
+  infoBoxes: [
+    {
+      title: "Infobox title",
+      description: "Infobox description",
+    },
+  ],
+}
+
+export const TwoInfoBoxes = Template.bind({})
+TwoInfoBoxes.args = {
+  sectionIdx: 0,
+  title: "Infobar title",
+  subtitle: "subtitle",
+  buttonLabel: "Button text",
+  buttonUrl: "https://google.com",
+  infoBoxes: [
+    {
+      title: "Infobox title",
+      description: "Infobox description",
+    },
+    {
+      title: "Infobox title",
+      description: "Infobox description",
+    },
+  ],
+}
+
+export const ThreeInfoBoxesLongText = Template.bind({})
```

Comment:
just curious, why are we developing both classic + next component simultaneously?